### DISCUSSION TITLE
Flappy test with model import

### DISCUSSION THREAD

**Discussion by keithschacht** [AllYourBot/hostedgpt#573] [General]
@drnic in main we just got this failed test:

```
Failure:
LanguageModel::ExportTest#test_import_from_file_with_only_new_models [test/models/language_model/export_test.rb:51]:
"LanguageModel.count" didn't change by 1, but by 0.
Expected: 31
  Actual: 30

bin/rails test test/models/language_model/export_test.rb:36

............................................................................................................
```

But I run it locally and it passed so I re-ran the failed test suite and it passed in main. I think we may have a flappy test. Any ideas?

**Comment by drnic** [comment]
This is indeed annoying and I’ve thought I’d figured it out “this must be the cause” half dozen times.

**Reply by drnic** [reply]
![IMG_1888](https://github.com/user-attachments/assets/9a40472f-ed97-43ab-b663-98eda97f89b5)

:)
